sim/shaman/elemental/elemental.go

```diff
@@ -71,7 +72,6 @@ func (eleShaman *ElementalShaman) Initialize() {
 	// Elemental Fury
 	eleShaman.AddStaticMod(core.SpellModConfig{
 		School:     core.SpellSchoolFire | core.SpellSchoolFrost | core.SpellSchoolNature,
-		ClassMask:  shaman.SpellMaskSearingTotem | shaman.SpellMaskMagmaTotem,
```

Comment:
You must provide a class mask here. Otherwise the mod will affect any crit caused by trinkets and other procs that can critical hit with a school damage, when they should not.

https://wago.tools/db2/SpellEffect?build=4.4.0.54339&filter[SpellID]=exact%3A60188&page=1 - The spell has excplicit clasks masks to not affect any non shaman spell